RSConnectDIOToSnap7/CodeBehind.cs

```diff
@@ -52,21 +52,21 @@ public class CodeBehind : SmartComponentCodeBehind
 		private S7Client client;
 		private bool bCanConnect = false;
 		private bool bPLC_AddrIsValid = false;
-		private bool[] bDI_AddressIsValid = { };
-		private bool[] bDO_AddressIsValid = { };
+        private bool[][] bDIO_AddressIsValid = new bool[2][];
+        public enum IO { Input,Output };
+        private double timePassed = 0;

```

Comment:
Please, respect indent.

Reply:
hmm, I don't know what happened to the intendation, i take a look on it

Reply:
The best is to show white space characters.

Reply:
in my version, the indentation is fine. maybe this is just a visual from github?

Reply:
there is also an automatic formatting function in visual studio, under edit->advanced->format document or selection. i give it a try.